style/javascript/sample.js

```diff
@@ -0,0 +1,7 @@
+object = { spacing: true }
+
+class Cat {
+  canBark() {
+    return false;
```

Comment:
I don't think automatic semicolon insertion has changed at all in ES6. So unless you're willing to live and die by its edge cases, then they have to stick around.